src/components/Terminal/Terminal.tsx

```diff
@@ -139,11 +141,21 @@ class Terminal extends React.Component<TerminalProps, {}> {
         e &&
           e.addEventListener('mouseup', () => {
             this.toggleChartOverlayHelper(false);
+            timeout = null;
           });
         // tslint:disable-next-line:no-unused-expression
         e &&
-          e.addEventListener('mousedown', () => {
+          e.addEventListener('mouseenter', () => {
```

Comment:
add 'mouseup' listener (e.g. to document.body) inside mousedown callback and remove overlay when it will be triggered. After end of 'mouseup' event remove its listener